canonical-web-and-design/ubuntu.com
# Update homepage

## Done
- Complete rebuilt the homepage content.
_Copy and layout has been approved by the PMs_

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit?ts=5f450201#heading=h.vmiyl9gud145)


## Issue / Card

Fixes #8171
